include/lvup.h

```diff
@@ -11,3 +11,39 @@ int GetUnitSpdGrowth(struct Unit * unit);
 int GetUnitLckGrowth(struct Unit * unit);
 int GetUnitDefGrowth(struct Unit * unit);
 int GetUnitResGrowth(struct Unit * unit);
+
+
+static inline int GetUnitBaseHpGrowth(struct Unit * unit)
```

Comment:
These API should hard to be called elsewhere, It is recommended not to add additional code for this